modules/virt-attaching-vm-secondary-network-cli.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc
+
+[id="virt-attaching-vm-secondary-network-cli_{context}"]
+= Attaching a virtual machine to a secondary network in the CLI
+
+Attach a virtual machine or virtual machine instance to a secondary network by adding a bridge to the `spec.domain.devices.interfaces` field and specifying the network attachment definition to the `spec.networks` field in the virtual machine configuration.
+
+This procedure uses a YAML file to demonstrate editing the configuration and applying the updated file to the cluster. You can alternatively use the `oc edit <object> <name>` command to edit an existing virtual machine or virtual machine instance.
+
+. Create or edit a new virtual machine or virtual machine instance configuration that you want to connect to the bridge network, for example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: VirtualMachine
+metadata:
+    name: example-vm
+spec:
```

Comment:
IIUC This path is correct for vmi, not vm.

Reply:
Updated the config